OneCommunityGlobal/HighestGoodNetworkApp
# Bhuvan - Unit test for UserProfileEdit components - SaveButton, ToggleSwitch and  TriStateToggleSwitch

## Main changes explained:

Added Five new files names 
1. [SaveButton.test.js](src/components/UserProfile/UserProfileEdit/__test__/SaveButton.test.js)
2. [ToggleSwitch.test.js](src/components/UserProfile/UserProfileEdit/__test__/ToggleSwitch.test.js)
3. [TriStateToggleSwitch.test.js](src/components/UserProfile/UserProfileEdit/__test__/TriStateToggleSwitch.test.js)

## How to test:
1. check into current branch 
 ```bash
   git checkout bhuvan-table
```
2. do `npm install` and `...` to run this PR locally
you can choose to run the files individually or as whole (jump to step 7)
3. run this command 
```bash 
npm run test:coverage D:/HGN/HighestGoodNetworkApp/src/components/UserProfile/UserProfileEdit/__test__/SaveButton.test.js
```
4. run this command 
```bash 
npm run test:coverage D:/HGN/HighestGoodNetworkApp/src/components/UserProfile/UserProfileEdit/__test__/ToggleSwitch.test.js
```
5. run this command 
```bash 
npm run test:coverage D:/HGN/HighestGoodNetworkApp/src/components/UserProfile/UserProfileEdit/__test__/TriStateToggleSwitch.test.js
```
![image](https://github.com/OneCommunityGlobal/HighestGoodNetworkApp/assets/67942070/be3d4eca-d924-42bf-b7fd-d3b75fca681d)
![image](https://github.com/OneCommunityGlobal/HighestGoodNetworkApp/assets/67942070/9c62378e-dd78-4549-bdc6-16ec7bdb3286)
